src/github.com/matrix-org/go-neb/util/util.go

```diff
@@ -0,0 +1,35 @@
+package util
+
+import (
+	"sort"
+)
+
+// Difference returns the elements that are only in the first list and
+// the elements that are only in the second. As a side-effect this sorts
+// the input lists in-place.
+func Difference(a, b []string) (onlyA, onlyB []string) {
```

Comment:
Probably not worth splitting this out to a separate file until there are two places it gets called.